src/styles/index.scss

```diff
@@ -1,3 +1,115 @@
-body {
-  margin: 0;
+$product-font-color: #060b35;
```

Comment:
Move variables to the separate file

Reply:
isn't fixed. You should create `variables.scss` file and move this variable there